packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -1,30 +1,33 @@
 const assert = require("assert");
 const { ChainId, Token, Pair, TokenAmount } = require("@uniswap/sdk");
-const { MedianizerPriceFeed } = require("./MedianizerPriceFeed");
-const { FallBackPriceFeed } = require("./FallBackPriceFeed");
-const { CryptoWatchPriceFeed } = require("./CryptoWatchPriceFeed");
-const { ForexDailyPriceFeed } = require("./ForexDailyPriceFeed");
-const { QuandlPriceFeed } = require("./QuandlPriceFeed");
-const { DefiPulsePriceFeed } = require("./DefiPulsePriceFeed");
-const { UniswapPriceFeed } = require("./UniswapPriceFeed");
-const { BalancerPriceFeed } = require("./BalancerPriceFeed");
-const { DominationFinancePriceFeed } = require("./DominationFinancePriceFeed");
-const { BasketSpreadPriceFeed } = require("./BasketSpreadPriceFeed");
-const { CoinMarketCapPriceFeed } = require("./CoinMarketCapPriceFeed");
-const { CoinGeckoPriceFeed } = require("./CoinGeckoPriceFeed");
-const { TraderMadePriceFeed } = require("./TraderMadePriceFeed");
-const { PriceFeedMockScaled } = require("./PriceFeedMockScaled");
-const { InvalidPriceFeedMock } = require("./InvalidPriceFeedMock");
 const { defaultConfigs } = require("./DefaultPriceFeedConfigs");
 const { getTruffleContract } = require("@uma/core");
-const { ExpressionPriceFeed, math, escapeSpecialCharacters } = require("./ExpressionPriceFeed");
-const { VaultPriceFeed } = require("./VaultPriceFeed");
-const { LPPriceFeed } = require("./LPPriceFeed");
 const { BlockFinder } = require("./utils");
 const { getPrecisionForIdentifier, PublicNetworks } = require("@uma/common");
-const { FundingRateMultiplierPriceFeed } = require("./FundingRateMultiplierPriceFeed");
 const { multicallAddressMap } = require("../helpers/multicall");
 
+// Price feed interfaces (sorted alphabetically)
```

Comment:
nice.

Reply:
Yeah tbh this whole file is getting out of hand, this was the minimum viable improvement and anything more felt like an overstep 😅

Reply:
ye. we do need to refactor much of this. it's a bit of a mess right now.